packages/snap/src/utils/version.ts

```diff
@@ -0,0 +1,38 @@
+import { SnapProvider } from "@metamask/snap-types";
+
+export const getMetamaskVersion = async (
+  wallet: SnapProvider
+): Promise<string> =>
+  (await wallet.request({
+    method: "web3_clientVersion",
+    params: [],
+  })) as string;
+
+export const isNewerVersion = (
```

Comment:
you should probably write some unit tests for this method

Reply:
it's a common method in snaps, probably the best way is to create a library for these utils and cover all that there.

Reply:
Why don't we use: https://www.npmjs.com/package/semver?